reposilite-backend/src/main/kotlin/com/reposilite/token/api/CreateAccessToken.kt

```diff
@@ -22,6 +22,11 @@ data class CreateAccessTokenRequest(
     val permissions: Set<AccessTokenPermission> = emptySet()
 )
 
+data class CreateAccessTokenRequestWithNoName(
```

Comment:
I think `CreateAccessTokenWithNoNameRequest` might be a better name, what do you think?